Doc/venv/lib/python3.9/site-packages/sphinx_rtd_theme/breadcrumbs.html

```diff
@@ -0,0 +1,77 @@
+{%- if meta is defined and meta is not none %}
+   {%- set check_meta = True %}
+{%- else %}
+   {%- set check_meta = False %}
+{%- endif %}
+
+{%- if check_meta and 'github_url' in meta %}
+   {%- set display_github = True %}
+{%- endif %}
+
+{%- if check_meta and 'bitbucket_url' in meta %}
+   {%- set display_bitbucket = True %}
+{%- endif %}
+
+{%- if check_meta and 'gitlab_url' in meta %}
+   {%- set display_gitlab = True %}
+{%- endif %}
+
+{%- set display_vcs_links = display_vcs_links if display_vcs_links is defined else True %}
+
+{#- Translators: This is an ARIA section label for page links, including previous/next page link and links to GitHub/GitLab/etc. -#}
+<div role="navigation" aria-label="{{ _('Page navigation') }}">
+  <ul class="wy-breadcrumbs">
+    {%- block breadcrumbs %}
+      <li><a href="{{ pathto('index') }}" class="icon icon-home" aria-label="Home"></a></li>
```

Comment:
Originally, w/o mod, this said
```
<a href="{{ pathto(master_doc) }}" 
```
which means the home button always takes you back to the ginormous TOC page (instead of the real "home" we think of at `index.html` from our beloved Pydocs).

So instead this is now:
```
<a href="{{ pathto('index') }}" 
```